### PR TITLE
KOGITO-8599: Fix Table External Component paging style in Dark Mode

### PR DESCRIPTION
**Pull request by kumaradityaraj**
The pagination section used to disappear when the screen size was decreased below 768px but now it comes down after the filter tab. This issue still needs to be addressed for light mode.
Screenshot from 2023-02-08 10-14-31

The pagination number which is in red used to be black in dark mode due to which it was invisible; in this PR it has been corrected and now its white in dark mode.
Screenshot from 2023-02-08 10-12-31

The updated code result is as in the below pic:-
Screenshot from 2023-02-08 10-20-37

### CODE REVIEW

**Comment by paulovmr** [comment]
Hi @kumaradityaraj , could you please add two images (or video, but I think image is good in this case) to the PR description showing the before and after the change? Thanks.

**Comment by kumaradityaraj** [comment]
> Hi @kumaradityaraj , could you please add two images (or video, but I think image is good in this case) to the PR description showing the before and after the change? Thanks.

The image has been updated. Thanks.

**Review by paulovmr — APPROVED** [review]

File: packages/dashbuilder-component-table/src/FilteredTable.tsx [hunk 1]
```diff
@@ -164,7 +164,7 @@ export const FilteredTable = (props: Props) => {
               onClear={() => onSearch("")}
             />
           </FlexItem>
-          <FlexItem align={{ default: "alignRight" }}>
+          <FlexItem align={{ default: "alignLeft" }}>
```
Comment (jesuino) [inline packages/dashbuilder-component-table/src/FilteredTable.tsx]: Could you please clarify why the position was changed?

File: packages/dashbuilder-component-table/static/index.css [hunk 2]
```diff
@@ -42,11 +42,38 @@
...
+  padding-left: 20px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.pf-l-flex {
+  justify-content: center;
+}
+
+.pf-m-align-left {
```
Comment (jesuino) [inline packages/dashbuilder-component-table/static/index.css]: same question, why changing the padding and alignment?

**Review by jesuino — CHANGES_REQUESTED** [review]
Changes looks good to me, thanks! I am just wondering about some additional CSS - could you please clarify it?

Thanks!

File: packages/dashbuilder-component-table/src/FilteredTable.tsx (hunk 1 above)
Comment (kumaradityaraj) [reply packages/dashbuilder-component-table/src/FilteredTable.tsx]: If we align it right then on changing the screen size less than 768px that particular div used to disappear; I tried alligning it left and then it worked.

File: packages/dashbuilder-component-table/static/index.css (hunk 2 above)
Comment (kumaradityaraj) [reply packages/dashbuilder-component-table/static/index.css]: In the small screen size, the UI was looking bad so I shifted the pagination part after the filter div and applied padding and aligned for smooth UI.

**Comment by jesuino** [comment]
Hello @kumaradityaraj 

Thanks for the clarification

Could we please keep only the color change for now?

On a later ticket we can work on the elements position. 

Basically it comes from patternfly[1] and we may have a patternfly way to fix sizing issues. We had to modify for dark mode because it is not supported for this component and it had to look like it is integrated into other components.

Thanks!

[1] https://www.patternfly.org/v4/components/table

**Comment by kumaradityaraj** [comment]
> Hello @kumaradityaraj
> 
> Thanks for the clarification
> 
> Could we please keep only the color change for now?
> 
> On a later ticket we can work on the elements position.
> 
> Basically it comes from patternfly[1] and we may have a patternfly way to fix sizing issues. We had to modify for dark mode because it is not supported for this component and it had to look like it is integrated into other components.
> 
> Thanks!
> 
> [1] https://www.patternfly.org/v4/components/table

The changes has been done. 
Thanks @jesuino

**Review by jesuino — APPROVED** [review]
Great work, thanks!

**Pull request merged**